src/components/molecules/Settings/Field/index.tsx

```diff
@@ -32,34 +32,34 @@ const Field: React.FC<Props> = ({ className, header, action, secondaryAction, bo
 const Wrapper = styled.div`
   display: flex;
   flex-direction: row;
-  justify-content: center;
-  width: 100%;
+  justify-content: space-between;
+  justify-items: center;
   color: ${colors.text.main};
 
   &:not(:last-child) {
-    margin-bottom: 40px;
+    margin-bottom: 24px;
   }
 `;
 
-const ActionArea = styled.div`
-  display: flex;
-  align-content: center;
-`;
-
 const Header = styled.div`
-  width: 220px;
-  margin-right: 20px;
+  width: 216px;
 `;
 
 const Content = styled.div`
-  flex-grow: 1;
+  max-height: 264px;
+  overflow: auto;
+  flex: 1;
+`;
+
+const ActionArea = styled.div`
```

Comment:
This could be a Flex component and you wouldn't need to create a styled component

Reply:
Also don't be afraid to updated components if they could be Flex (or Text or another one of our atoms components). There are some stragglers from before we had those components!

Reply:
and here, because the Flex-component don't have justify-items property, I think it's better not to use Flex-component ..